src/models/registry.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity, CreateDateColumn, UpdateDateColumn, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { User } from './user';
+import { Course } from './course';
+
+@Entity()
+export class Registry {
+  @PrimaryGeneratedColumn() id: number;
+
+  @ManyToOne(_ => User, (user: User) => user.registries, { nullable: true })
+  user: User | number;
+
+  @ManyToOne(_ => Course, (course: Course) => course.registries, { nullable: true })
+  course: Course | number;
+
+  @Column({ name: 'type', enum: ['mentor', 'mentee'] })
```

Comment:
please don't use `enum` type. We discovered some issues related to integration with other systems (Kafka) and moving way from them.